### PR TITLE
📜 adds start page info for teachers

### PR DESCRIPTION
**Pull request by SabinaChita**
Added compact information about using Hedy as a teacher.

### CODE REVIEW

**Comment by Felienne** [comment]
Maybe also something like:

" the first 5 things you will want to try are..."

**Comment by Felienne** [comment]
Looks good @SabinaChita!

Do we want to add som extra info to "we are here to help"? (email, Discord? office hours?)

**Comment by Felienne** [comment]
> I actually did add discord plus email, you cannot see it? :-(

this is what I see:

image
 Maybe you did not yet commit or push?

**Comment by Felienne** [comment]
I think you just forgot to commit or push in GitHb Desktop (nothing stupid)

File: requirements.txt [hunk 1]
```diff
@@ -32,4 +32,9 @@ jinja-partials==0.1.1
 hypothesis>=6.75.3
 tqdm==4.65.0
 pytest-xdist==3.3.1
-email-validator==2.1.0.post1
\ No newline at end of file
+email-validator==2.1.0.post1
```
Comment (Felienne) [inline requirements.txt:35]: Hi @SabinaChita!

I am not sure why these are added, do you know? Did you add this on purpose?

**Comment by SabinaChita** [comment]
Hi Felienne,

No I did not add this! I don’t know what the line means.

Met vriendelijke groet,

Dr. Sabina Chiță
Docent, mentor en studiebegeleider, VASVU
Student- & Onderwijszaken

[Logo]

Werkdagen: ma, di, do | T 31 20 59 84426| ***@***.***
Bezoekadres: https://vu.nl/nl/onderwijs/meer-over/voorbereidend-jaar-anderstalige-studenten-vasvu
Werkdagen: vr | ***@***.***
Bezoekadres: https://www.hedycode.com/
Postadres: De Boelelaan 1105, 1081 HV Amsterdam | E-mail disclaimer<https://vu.nl/nl/over-de-vu/meer-over/vu-website> | www.vu.nl<https://vu.nl/nl/>

[Facebook]<https://www.facebook.com/vuamsterdam/>    [Instagram] <https://www.instagram.com/vuamsterdam/>     [LinkedIn] <https://www.linkedin.com/school/5570/admin/>     [Twitter] <http://twitter.com/#!/vuamsterdam>     [YouTube] <https://www.youtube.com/user/vrijeuniversiteit>

________________________________

File: requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Flask==2.3.2
-Werkzeug==3.0.1
+Werkzeug~=2.3.7
```
Comment (Felienne) [inline requirements.txt:2]:
```suggestion
Werkzeug~=3.0.1
```

File: requirements.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Flask==2.3.2
-Werkzeug==3.0.1
+Werkzeug~=3.0.1
```
Comment (Felienne) [inline requirements.txt:2]:
```suggestion
Werkzeug==3.0.1
```

**Comment by Felienne** [comment]
> Hi @Felienne, no I did not add this! I don’t know what the line means.

ok I guess you ran an extraction by accident, I removed the lines

**Review by Felienne — APPROVED** [review]
Let's merge this for now, Giselle will pick this up in redesigning too

File: content/quizzes/en.yaml [hunk 4]
```diff
@@ -4158,5 +4158,3 @@ levels:
             -   option: There is an indentation mistake in the last line.
                 feedback: Nope.
             hint: Read the code carefully.
```
Comment (Felienne) [inline content/quizzes/en.yaml:4160]:
```suggestion
            hint: Read the code carefully.
            correct_answer: C
            question_score: '10'
```

**Pull request merged**